.github/workflows/main.yml

```diff
@@ -39,6 +39,7 @@ jobs:
           # scp-action doesn't obey working-directory, runs at repo root
           source: "build/"
           target: "/var/www/html/public_html/"
+          strip_components: 1
```

Comment:
the action worked but everything was in the build folder, what this will do is only copy the children over and not the parent build folder